documentation/en/default-lotus-config.toml

```diff
@@ -1,380 +0,0 @@
-[API]
```

Comment:
you're going to need to replicate this too, but it should be pretty easy, the `lotus config default` is fairly simple: https://github.com/filecoin-project/lotus/blob/07f2f6990b55b4a4b93ee94609172c85335e92fd/cmd/lotus/config.go#L32-L40